test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tests
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/pkg/common"
+	gce "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/pkg/gce-cloud-provider/compute"
+	testutils "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/e2e/utils"
+	remote "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/remote"
+
+	csi "github.com/container-storage-interface/spec/lib/go/csi/v0"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	testNamePrefix = "gcepd-csi-e2e-"
+
+	defaultSizeGb    int64 = 5
+	readyState             = "READY"
+	standardDiskType       = "pd-standard"
+	ssdDiskType            = "pd-ssd"
+)
+
+var _ = Describe("GCFS CSI Driver", func() {
+
+	It("Should create->attach->stage->mount volume and check if it is writable, then unmount->unstage->detach->delete and check disk is deleted", func() {
+		Expect(testInstances).NotTo(BeEmpty())
+		testContext, err := testutils.GCEClientAndDriverSetup(testInstances[0])
```

Comment:
Is the purpose to run the GCE PD driver tests or to run the GCFS driver?

Reply:
If the purpose is to run the GCFS driver, then try incorporating the test I wrote using the old framework here: https://github.com/msau42/gcp-filestore-csi-driver/blob/e2e-test/test/e2e/filestore_test.go